src/ast/InterfaceDecl.ts

```diff
@@ -31,10 +31,8 @@ export class InterfaceDecl extends Content {
         this.comments = new CommentDecl();
         this.comments.parse(context);
 
-        this.fields = [];
-        this.fields.forEach((field) => {
-            field.parse(context);
-        });
+        this.fieldDecl = new FieldDecl();
+        this.fieldDecl.parse(context);
 
```

Comment:
I think this still needs to be an array, since we can have multiple field decls (e.g. 1 for private, 1 for public)

Reply:
I don't think so - at least not in `InterfaceDecl`, since they can't have access modifiers in the class.

Reply:
Ah I see- good point!